internal/metadataproviders/system/metadata.go

```diff
@@ -63,6 +63,9 @@ type Provider interface {
 
 	// HostArch returns the host architecture
 	HostArch() (string, error)
+
+	// HostIPv4Addresses returns the host's IPv4 interfaces
```

Comment:
I guess this docstring needs to be updated?

Reply:
Indeed, fixed!